dash/backend/src/exception-filters/http-exception.filter.ts

```diff
@@ -17,13 +17,27 @@ export class HttpExceptionFilter implements ExceptionFilter {
     const ctx = host.switchToHttp();
     const request = ctx.getRequest<Request>();
     const response = ctx.getResponse<Response>();
-    
+
     // build response
     const status = exception.getStatus();
 
     let errorMessage = exception.message;
 
-    if (exception instanceof BadRequestException && typeof exception.getResponse === 'function' && 
+    const respFromException: any = exception.getResponse();
```

Comment:
Why was this changed?

Reply:
This filter stripped out any extra data provided by an error.

Reply:
Recheck :)